lib/scanner/index.js

```diff
@@ -30,13 +40,22 @@ class Scanner {
   }
 
   /**
-   * Start scanning.
+   * Perform a scan.
    *
    * @public
+   * @param  {Number} duration
+   * @return {Promise}
    */
-  start() {
-    debug('start');
-    this.cycle();
+  scan(duration) {
```

Comment:
The user may still want to stop a scan early? We could keep the start/stop API, but it will just mean we have to move the timing logic into app.js.